src/xrGame/script_game_object2.cpp

```diff
@@ -153,7 +153,7 @@ void CScriptGameObject::Hit(CScriptHit* tpLuaHit)
     SHit HS;
     HS.GenHeader(GE_HIT, object().ID()); //	object().u_EventGen(P,GE_HIT,object().ID());
     THROW2(tLuaHit.m_tpDraftsman,
-        "Where is hit initiator??!"); //	THROW2			(tLuaHit.m_tpDraftsman,"Where is hit initiator??!");
+        "Where is hit initiator?!"); //	THROW2			(tLuaHit.m_tpDraftsman,"Where is hit initiator?!");
```

Comment:
Don't touch the GSC comments.

Reply:
`??!` is being treated as trigraphs and generates warning because it's deprecated.

Reply:
Oh...

Reply:
Please, return original GSC EMOTION and just disable ridiculous warning via key.